extra/export_model.py

```diff
@@ -125,24 +127,25 @@ def export_model_webgpu(functions, statements, bufs, bufs_to_save, weight_names)
 }}
   """ + f"\n\nconst loadNet = async (device) => {{ return await fetch('net.safetensors').then(x => x.arrayBuffer()).then(x => setupNet(device, new Uint8Array(x))); }}"
 
-def export_model(model, input:Tensor, target:str):
+def export_model(model, inputs:List[Tensor], target:str):
```

Comment:
switch the ordering of `inputs` and `target` to make the api cleaner with a variable number arg list

Reply:
just fixed this